docs/ui/foundations/icons.tsx

```diff
@@ -0,0 +1,198 @@
+import { iconSize, palette, theme } from '@expo/styleguide';
+import React from 'react';
+import { Clipboard } from 'react-feather';
+
+export type IconProps = {
```

Comment:
@jonsamp any chance to get this exported from styleguide instead? We had to add it to explicitly mention only icons are supported as property of something like the Callout/Blockquote.